src/Sarif.Multitool/QueryCommand.cs

```diff
@@ -21,11 +22,13 @@ public QueryCommand(IFileSystem fileSystem = null)
 
         public int Run(QueryOptions options)
         {
+            bool valid = DriverUtilities.ReportWhetherOutputFileCanBeCreated(options.OutputFilePath, options.Force, _fileSystem);
```

Comment:
>ReportWhetherOutputFileCanBeCreated [](start = 41, length = 35)

Perhaps move this into RunWithoutCatch so it's consistent with PageCommand.

Reply:
Good, done.

---
In reply to: [312280805](https://github.com/microsoft/sarif-sdk/pull/1625#discussion_r312280805) [](ancestors = 312280805)